test/blocks/global-footer/global-footer.test.js

```diff
@@ -0,0 +1,247 @@
+import { expect } from '@esm-bundle/chai';
+import sinon, { stub } from 'sinon';
+import {
+  allElementsVisible,
+  visibleSelectorsDesktop,
+  visibleSelectorsMobile,
+  containerSelector,
+  createFullGlobalFooter,
+  insertDummyElementOnTop,
+  isElementVisible,
+  waitForFooterToDecorate,
+  allSelectors,
+} from './test-utilities.js';
+import baseFooter from './mocks/base-footer.js';
+import fetchedFooter from './mocks/fetched-footer.js';
+import icons from './mocks/icons.js';
+import { mockRes } from '../global-navigation/test-utilities.js';
+import placeholders from './mocks/placeholders.js';
+
+const originalFetch = window.fetch;
```

Comment:
I know we use this approach in the global-navigation testing as well, but couldn't we just use `.restore()` in the `afterEach` call?

Reply:
Done 👍 Yes, it's cleaner with `.restore()` in the `afterEach`